nearlib/test/integration.test.js

```diff
@@ -136,7 +136,8 @@ describe('with deployed contract', () => {
     test('can get assert message from method result', async () => {
         await expect(contract.triggerAssert()).rejects.toThrow(/Transaction .+ failed.+expected to fail/);
         expect(logs).toEqual([`[${contractName}]: LOG: log before assert`,
-            `[${contractName}]: ABORT: "expected to fail" filename: "main.ts" line: 35 col: 2`]);
+            `[${contractName}]: ABORT: "expected to fail" filename: "main.ts" line: 44 col: 2`,
```

Comment:
This should not be changed. Rerun `npm install && npm run build` int tests/hello

Reply:
Why not? I've changed the main.ts and added morelines before

Reply:
I see